jose.pyx

```diff
@@ -0,0 +1,141 @@
+from libc.stdlib cimport free
+cimport jansson
+cimport jose
+
+import json
+
+def jwk_generate(jwk):
+    cdef jansson.json_t *cjwk = NULL
+    cdef char *ret = NULL
+
+    assert isinstance(jwk, dict)
+
+    try:
+        cjwk = jansson.json_loads(json.dumps(jwk).encode(u"UTF-8"), 0, NULL)
+        assert cjwk
+
+        assert jose.jose_jwk_generate(cjwk)
```

Comment:
Let's use them for now just to get a rough version of the API working. Then you can see what the API should look like. Then I'm happy to change it once we have the big picture.